src/PhpSpreadsheet/Shared/Date.php

```diff
@@ -206,11 +206,12 @@ public static function excelToDateTimeObject($excelTimestamp, $timeZone = null)
 
     /**
      * Convert a MS serialized datetime value from Excel to a unix timestamp.
+     * The use of Unix timestamps, and therefore this function, is discouraged.
```

Comment:
While the commit message explain the situation well, I'd expect to have a least a succinct explanation in the code itself, as it will most likely be the only thing that end-users see

Reply:
Done.